phys/module_diag_solar.F

```diff
@@ -88,6 +90,13 @@ SUBROUTINE solar_diag (rho, dz8w, ph, phb, cldfrac3d, coszen, swdnb, swdnt, &
                clrnidx(i, j) = MISSING
              end if
 
+             !!! ACCUMULATED GHI
+             if (is_first_timestep) then
+               ghi_accum(i, j) = 0.0
+             else
+               ghi_accum(i, j) = ghi_accum(i, j) + swdnb(i, j) * dt
+             end if
+
```

Comment:
@twjuliano @weiwangncar 
Tim,
These "if this is the first time step, do THIS" statements are sometimes dangerous for OpenMP. Can the first part of this test, the initialization to zero, be placed in module_physics_init?

Reply:
@davegill Sure, that's no problem. Is it OK if I add it after the Fitch wind farm initialization and right before the call to `ra_init`?

Reply:
@twjuliano 
Tim,
Pass this into the ra_init routine (right after the Fitch routine). The call to phy_init is from dyn_em/start_em.F, all variables are explicitly passed in. At the top of the ra_init is a "start_of_simulation" IF test.

Reply:
@davegill Done